File: labrabota7-gr13-2-MoonLory/Task2_lab7/Task2_lab7/ViewController.m
//
//  ViewController.m
//  Task2_lab7
//
//  Created by Admin on 18/04/2020.
//  Copyright © 2020 Admin. All rights reserved.
//

#import "ViewController.h"
#import "CityInfoRetriever.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UILabel *temperatureLabel;
@property (weak, nonatomic) IBOutlet UIButton *temperatureButton;
@property (weak, nonatomic) IBOutlet UITextField *cityTextField;
@property (weak, nonatomic) IBOutlet UIImageView *coatOfArmsView;
@property (weak, nonatomic) IBOutlet UIImageView *flagView;
@property (weak, nonatomic) IBOutlet UILabel *countryLabel;

@end

@implementation ViewController
- (IBAction) temperatureButtonTouched: (id) sender {
    @try {
        NSNumber *temp = [CityInfoRetriever getTemperatureByCity: self.cityTextField.text];
        [self.temperatureLabel setText: [temp.description stringByAppendingString: @" C"]];
        [self.temperatureLabel setTextColor: [CityInfoRetriever getColorByTemperature: temp]];
    
        [self.countryLabel setText: [@"Population " stringByAppendingString: [CityInfoRetriever getCountryByCity:   self.cityTextField.text]]];
    
        [self.flagView setImage: [CityInfoRetriever getMayorByCity: self.cityTextField.text]];
    }
    @catch (NSException *ex) {
        [self.temperatureLabel setText: @""];
        [self.countryLabel setText: @""];
        [self.flagView setImage: nil];
    }
}

- (void)viewDidLoad {
  [super viewDidLoad];
    // Do any additional setup after loading the view.
}


@end

File: labrabota7-gr13-2-MoonLory/Task4_lab7/ViewController.swift
//
//  ViewController.swift
//  Task4_lab7
//
//  Created by Admin on 19/04/2020.
//  Copyright © 2020 Admin. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var num1: UITextField!
    @IBOutlet weak var num2: UITextField!
    @IBOutlet weak var textans: UILabel!
    @IBOutlet weak var perekl: UISegmentedControl!
    @IBAction func ans(_ sender: Any) {
        
        
        let n1=Double(num1.text ?? "0") ?? 0.0;
        let n2=Double(num2.text ?? "0") ?? 0.0;
        
        if (perekl.selectedSegmentIndex==0){
            textans.text="Ответ: "+String(n1+n2);
            
        }
        else{
            textans.text="Ответ: "+String(n1-n2);
        }
        
        
    }
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }


}

